person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unkown', _parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    of_age?
+  end
+
```

Comment:
The method `can_use_services?` should return true if a person is of age or **if they have permission from parents**.

- Kindly include the `@parent_permission` variable in your return statement.